### PR TITLE
Librosa

### PR DESCRIPTION
**Pull request by NeuroForLunch**

### CODE REVIEW

File: docsets/librosa/docset.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "librosa",
+    "version": "0.8.1",
+    "archive": "Sample.tgz",
```
Comment (Kapeli) [inline docsets/librosa/docset.json:4]: Please change this to `librosa.tgz`

**Comment by Kapeli** [comment]
`icon.png` needs to be `16x16` and `icon@2x.png` needs to be `32x32` in size.

**Comment by Kapeli** [comment]
Why did you change the folder name to `librosa`? Changing the folder name will cause for duplicate entries to be shown to users that already installed the old docset.

File: docsets/librosa/docset.json (hunk 1 above)
Comment (NeuroForLunch) [reply docsets/librosa/docset.json]: fixed.

**Comment by NeuroForLunch** [comment]
> Why did you change the folder name to `librosa`? Changing the folder name will cause for duplicate entries to be shown to users that already installed the old docset.

The library is actually called librosa or Librosa and not libROSA. I tried to change it back to libROSA on git but it doesn't seem to pick up the capitalization change.

**Comment by Kapeli** [comment]
Use this:

```
git mv librosa temp
git mv temp LibROSA
```

The folder name is used just as the internal identifier in Dash. You can change the name users see in `docset.json`.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**